client/ayon_maya/plugins/create/create_model.py

```diff
@@ -39,5 +39,9 @@ def get_instance_attr_defs(self):
                     placeholder="attr1, attr2"),
             TextDef("attrPrefix",
                     label="Custom Attributes Prefix",
-                    placeholder="prefix1, prefix2")
+                    placeholder="prefix1, prefix2"),
+            BoolDef("include_shaders",
```

Comment:
@antirotor should this be `"include_shaders"` or `"includeShaders"`? I stuck with snake case to match your PR, but it seems that camelCase is the convention for the other settings.

Reply:
In this case the snake case is good. Thanks for checking! :)